src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3VettingHelper.java

```diff
@@ -59,13 +67,73 @@ public static List<String> populateRespondentDynamicList(CaseData caseData) {
         }
 
         List<DynamicValueType> dynamicRespondentList =
-                DynamicListHelper.createDynamicRespondentName(respondentCollection);
+            DynamicListHelper.createDynamicRespondentName(respondentCollection);
         DynamicFixedListType dynamicFixedListType = new DynamicFixedListType();
         dynamicFixedListType.setListItems(dynamicRespondentList);
         caseData.setEt3ChooseRespondent(dynamicFixedListType);
         return new ArrayList<>();
     }
 
+    public static void getRespondentNameAndAddress(CaseData caseData) {
```

Comment:
Need JavaDoc on public methods.  A prime example here is that the method name doesn't describe what is actually happening and what happens if an address isn't found, etc...